packages/terra-consumer-nav/src/Nav.scss

```diff
@@ -3,10 +3,12 @@
 
 :local {
   .nav-container {
+    background: inherit;
     z-index: 1000;
```

Comment:
this was a typo on my end and has a weird interaction with your help modal on mobile view, change this to 100 so it's behind your modal/overlay.